docs/api-docs/store-management/shipping/use-a-shipping-provider.md

```diff
@@ -0,0 +1,375 @@
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example DELETE request with X-Auth-Token header" lineNumbers
+DELETE https://api.bigcommerce.com/stores/{store_hash}/v2/shipping/carrier/connection
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "carrier_id": "example_carrier"
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example DELETE response" lineNumbers
+No content
+```
+
+<!-- type: tab-end -->
+
+
+## Manage shipping methods
+
+After you connect a carrier, you must define and enable a shipping method for the carrier in a shipping zone. You can define and enable multiple shipping methods for a carrier for any zone.    
+
+![Enabled vs Connected Carrier](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Enabled%20vs%20Connected%20Carrier.png 'Enabled versus Connected Carrier. USPS is enabled, while FedEx is only connected.')  
+
+
+### Create a shipping method
+
+You can define and enable a method for the connected carrier in any shipping zone by using the [Create a shipping method](/api-reference/store-management/shipping-api/shipping-method/createashippingmethod) endpoint. Specify the shipping zone in the path using the zone ID (`id`) from the [Get all shipping zones](/api-reference/store-management/shipping-api/shipping-zones/getallshippingzones) endpoint. The shipping carrier’s ID is required. 
+
+<!--
+type: tab
+title: Request
+-->
+
+```json title="Example POST request with X-Auth-Token header" lineNumbers
+POST https://api.bigcommerce.com/stores/{store_hash}/v2/shipping/zones/{zone_id}/methods
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "name": "USPS",
+  "type": "endicia",
+  "settings": {
+      "carrier_options": {
+          "show_transit_time": "1",
+          "packaging_type": "FlatRateLegalEnvelope",
+          "delivery_services": [
+              "PriorityExpress",
+              "ParcelSelect",
+              "MediaMail"
+          ]
+      }
+  },
+  "enabled": true,
+  "handling_fees": {
+      "fixed_surcharge": "0"
+  }
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example POST response" lineNumbers
+{
+   "id": "29",
+   "name": "USPS",
+   "type": "endicia",
+   "settings": {
+      "carrier_options": {
+         "show_transit_time": "1",
+         "packaging": "FlatRateLegalEnvelope",
+         "packaging_type": "FlatRateLegalEnvelope",
+         "delivery_services": [
+            "PriorityExpress",
+            "ParcelSelect",
+            "MediaMail"
+         ]
+      }
+   },
+   "enabled": "true",
+   "handling_fees": {
+      "fixed_surcharge": "0"
+   },
+   "is_fallback": "false"
+}
+```
+
+<!-- type: tab-end -->
+
+To see a full list of available `carrier_options`values for a carrier, see the [Create a shipping method](/api-reference/store-management/shipping-api/shipping-method/createashippingmethod) reference.
```

Comment:
We won't have documentation for all third-party apps. We may want to refer devs to the shipping provider to understand their options.